api/_hyperfunctions/count_min_sketch/count_min_sketch.md

```diff
@@ -0,0 +1,48 @@
+---
+api_name: count_min_sketch()
+excerpt: Aggregate data into a `CountMinSketch` for approximate counting
+topics: [hyperfunctions]
+api:
+  license: community
+  type: function
+  toolkit: true
+  experimental: true
+  version:
+    experimental: 1.8.0
+hyperfunction:
+  family: frequency analysis
+  type: aggregate
+  aggregates:
+    - count_min_sketch()
+api_details:
+  summary: |
+    Aggregate data into a `CountMinSketch` object, which you can then use to estimate the number of times a given item appears in a column.
+    The sketch produces a biased estimator of frequency.
+    It might overestimate the item count, but it won't underestimate.
```

Comment:
```suggestion
    The sketch produces a biased estimator of frequency; it might overestimate the item count, but it can't underestimate.
```

There's a subtle difference between "can't" and "won't" here. I feel like "won't" says "it shouldn't" which still implies that it *could*.